examples/config/zfs.yaml

```diff
@@ -0,0 +1,36 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: local-path-config
+  namespace: local-path-storage
+data:
+  config.json: |-
+    {
+      "nodePathMap":[{
+        "node":"DEFAULT_PATH_FOR_NON_LISTED_NODES",
+        "paths":["/tank/"]
+      }]
+    }
+  setup: |-
+    #!/bin/sh
+    zfs create -o quota=${VOL_SIZE_BYTES} ${VOL_DIR:1}
```

Comment:
IIUC,
`${VOL_DIR:1}` is the zfs filesystem name, like `tank/pvc-xxxxx`.

If the zfs pool mountpoint (paths) is `/mnt/tank` and the pool name `tank`, the `zfs create ..` will fail.

Suggest adding a comment or readme doc describing how to set the paths/pool mountpoint correctly.